astropy/visualization/mpl_normalize.py

```diff
@@ -4,12 +4,18 @@
 
 from __future__ import division, print_function
 
+from distutils.version import LooseVersion
+
 import numpy as np
 from numpy import ma
 
 try:
+    import matplotlib
+    MATPLOTLIB_INSTALLED = True
```

Comment:
Not that it matters, but maybe use `HAS_MATPLOTLIB`--more consistent with the style used elsewhere.